src/main/java/org/llorllale/cactoos/matchers/MatcherEnvelope.java

```diff
@@ -40,7 +40,7 @@
  * Matcher Envelope.
  * @param <T> The type of the Matcher.
  * @since 1.0.0
- * @todo #75:30min Refactor other matchers to extend MatcherEnvelope.
+ * @todo #94:30min Refactor other matchers to extend MatcherEnvelope.
  *  If you do not know how to do it please refer to InputHasContent
  *  class as the example.
  */
```

Comment:
Constructor `MatcherEnvelope(final Matcher<T> origin...` is never unit tested directly. So it either needs to be unit tested using public/protected call or marked as private.

Reply:
Added a `@todo` to write tests for `MatcherEnvelope` (currently it's tested only indirectly).